tools/osv-scanner/cmd/scanner.go

```diff
@@ -103,6 +103,44 @@ func scanGit(repoDir string) (*osv.Query, error) {
 	return osv.MakeCommitRequest(commit), nil
 }
 
+type DockerPackageVersion struct {
```

Comment:
Can you add some comments here? See the "Types" section on https://go.dev/doc/comment